src/verifiers/documentStatus/tokenRegistryStatus/openAttestationEthereumTokenRegistryStatus.test.ts

```diff
@@ -155,65 +157,67 @@ describe("openAttestationEthereumTokenRegistryStatus", () => {
+            "reason": Object {
+              "code": 1,
+              "codeString": "DOCUMENT_NOT_MINTED",
+              "message": "Token registry 0x0000000000000000000000000000000000000000 is not found",
+            },
+            "status": "INVALID",
+            "type": "DOCUMENT_STATUS",
           },
-          status: "INVALID",
-        },
-      ]);
+        ]
+      `);
     });
     it("should return an invalid fragment when document with token registry has not been minted", async () => {
       const fragment = await verify(documentRopstenNotIssuedWithTokenRegistry, {
         network: "ropsten",
       });
-      expect(fragment).toStrictEqual([
-        {
-          name: "OpenAttestationEthereumTokenRegistryStatus",
-          type: "DOCUMENT_STATUS",
-          data: {
-            details: [
-              {
-                address: "0xb53499ee758352fAdDfCed863d9ac35C809E2F20",
-                minted: false,
-                reason: {
-                  code: 1,
-                  codeString: "DOCUMENT_NOT_MINTED",
-                  message:
-                    "Document 0x693c86fbb8f75ac56f865f5b3100e545875f2154b3749bdcf448c874a1d67ef3 has not been issued under contract 0xb53499ee758352fAdDfCed863d9ac35C809E2F20",
+      expect(fragment).toMatchInlineSnapshot(`
+        Array [
+          Object {
+            "data": Object {
+              "details": Array [
+                Object {
+                  "address": "0xb53499ee758352fAdDfCed863d9ac35C809E2F20",
+                  "minted": false,
+                  "reason": Object {
+                    "code": 1,
+                    "codeString": "DOCUMENT_NOT_MINTED",
+                    "message": "Document 0x693c86fbb8f75ac56f865f5b3100e545875f2154b3749bdcf448c874a1d67ef3 has not been issued under contract 0xb53499ee758352fAdDfCed863d9ac35C809E2F20",
+                  },
                 },
-              },
-            ],
-            mintedOnAll: false,
-          },
-          reason: {
-            code: 1,
-            codeString: "DOCUMENT_NOT_MINTED",
-            message:
-              "Document 0x693c86fbb8f75ac56f865f5b3100e545875f2154b3749bdcf448c874a1d67ef3 has not been issued under contract 0xb53499ee758352fAdDfCed863d9ac35C809E2F20",
+              ],
+              "mintedOnAll": false,
+            },
+            "name": "OpenAttestationEthereumTokenRegistryStatus",
+            "reason": Object {
+              "code": 1,
+              "codeString": "DOCUMENT_NOT_MINTED",
+              "message": "Document 0x693c86fbb8f75ac56f865f5b3100e545875f2154b3749bdcf448c874a1d67ef3 has not been issued under contract 0xb53499ee758352fAdDfCed863d9ac35C809E2F20",
+            },
+            "status": "INVALID",
+            "type": "DOCUMENT_STATUS",
           },
-          status: "INVALID",
-        },
-      ]);
+        ]
+      `);
     });
     it("should return a valid fragment when document with token registry has been minted", async () => {
```

Comment:
the description is wrong

Reply:
Why?
should return a valid fragment when `document with token registry` has been minted

Reply:
raaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa FUCKING GITHUB REVIEW MODE